composer.json

```diff
@@ -14,10 +14,11 @@
 	},
 	"require-dev": {
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-		"squizlabs/php_codesniffer": "3.6.0",
+		"squizlabs/php_codesniffer": "^3.7.1",
 		"wp-coding-standards/wpcs": "~2.3.0",
 		"phpcompatibility/phpcompatibility-wp": "~2.1.3",
-		"yoast/phpunit-polyfills": "^1.0.1"
+		"yoast/phpunit-polyfills": "^1.0.1",
+		"slevomat/coding-standard": "^8.0"
```

Comment:
Ah, this sniff requires PHP 7.2+. Per [GHA log](https://github.com/WordPress/wordpress-develop/actions/runs/4985421270/jobs/8925010228?pr=4457#step:6:141):

```
Error: Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - slevomat/coding-standard[8.0.0, ..., 8.12.0] require php ^7.2 || ^8.0 -> your php version (5.6.40) does not satisfy that requirement.
    - Root composer.json requires slevomat/coding-standard ^8.0 -> satisfiable by slevomat/coding-standard[8.0.0, ..., 8.12.0].


  Problem 1
    - slevomat/coding-standard[8.0.0, ..., 8.12.0] require php ^7.2 || ^8.0 -> your php version (5.6.40) does not satisfy that requirement.
    - Root composer.json requires slevomat/coding-standard ^8.0 -> satisfiable by slevomat/coding-standard[8.0.0, ..., 8.12.0].
```